test/Microsoft.AspNet.Razor.Runtime.Precompilation.Test/PrecompilationTagHelperDescriptorFactoryTest.cs

```diff
@@ -27,5 +27,54 @@ public override ITypeInfo GetTypeInfo(Type tagHelperType)
             return Assert.Single(typeResolver.GetExportedTypes(CompilationUtility.GeneratedAssemblyName),
                 generatedType => string.Equals(generatedType.FullName, tagHelperType.FullName, StringComparison.Ordinal));
         }
+
+        [Theory]
+        [MemberData(nameof(TagHelperWithPrefixData))]
+        public void CreateDescriptors_WithPrefixes_ReturnsExpectedAttributeDescriptors(
```

Comment:
We were. I had to copy the test over here so I could customize the behavior (the CoreCLR sanitization) for precompilation.